charts/sourcegraph/templates/searcher/searcher.StatefulSet.yaml

```diff
@@ -89,7 +81,7 @@ spec:
           timeoutSeconds: 5
         volumeMounts:
         - mountPath: /mnt/cache
-          name: cache-ssd
+          name: cache
```

Comment:
Should we keep cache-ssd as the volume name? We are also using cache-ssd in the kustomize deployment. What do you think?